porches/generic/site/archive.php

```diff
@@ -50,15 +50,15 @@ public function dt_custom_dir_attr( $lang ){
     }
 
     public function dt_magic_url_base_allowed_js( $allowed_js ) {
-        return P4_Ramadan_Porch_Landing_Enqueue::load_allowed_scripts();
+        return DT_Generic_Porch_Landing_Enqueue::load_allowed_scripts();
```

Comment:
@squigglybob this does not exist?

Reply:
DT_GenericPorch_Landing_Enqueue does tho

Reply:
ah, good spot, I think when I merged thinks must have gone awry. I'll get that switched over to proper camel casing

Reply:
@corsacca should be fixed